src/GraphQL/Utilities/NameValidator.cs

```diff
@@ -12,14 +12,14 @@ public static class NameValidator
         /// </summary>
         /// <param name="name">GraphQL name.</param>
         /// <param name="type">Type of element: field, type, argument, enum.</param>
-        public static void ValidateName(string name, NamedElement type) => GlobalSwitches.Validation(name, type);
+        public static void ValidateName(string name, NamedElement type) => GlobalSwitches.NameValidation(name, type);
 
         /// <summary>
         /// Validates a specified name during schema initialization.
         /// </summary>
         /// <param name="name">GraphQL name.</param>
         /// <param name="type">Type of element: field, type, argument, enum.</param>
-        public static void ValidateNameOnSchemaInitialize(string name, NamedElement type) => GlobalSwitches.ValidationOnSchemaInitialize(name, type);
+        public static void ValidateNameOnSchemaInitialize(string name, NamedElement type) => ValidateDefault(name, type);
```

Comment:
Remove ? Only two usages from `private` method.

Reply:
I left it there because then it was an obvious what the intent of the code was versus calling `ValidateDefault` directly.  We could make it `internal` if you like.  I'm sure JIT would optimize out this call so there shouldn't be a performance difference.

Reply:
OK with internal.